lib/haproxy/haproxy.go

```diff
@@ -18,8 +18,10 @@ import (
 func Reload(clusters []data.Cluster, uid, gid uint32) error {
 	config :=
 		"global\n" +
-		"    daemon\n\n" +
+		"    daemon\n" +
```

Comment:
This might be easier to express as a [string literal](https://golang.org/ref/spec#String_literals).